runtime/interpreter/value.go

```diff
@@ -6419,7 +6411,7 @@ func (Int128Value) SetMember(_ *Interpreter, _ func() LocationRange, _ string, _
 }
 
 func (v Int128Value) ToBigEndianBytes() []byte {
-	return SignedBigIntToBigEndianBytes(v.BigInt)
+	return SignedBigIntToSizedBigEndianBytes(v.BigInt, 128/8)
```

Comment:
Maybe define constants for the magic numbers. `sema` already defines some related constants e.g see `Int128Type*`.